modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -0,0 +1,663 @@
+                                ' -nodaf -ise'
+                       },
+        -max_retry_count => 0,
+        -rc_name => '8GB',
+        -flow_into => {
+                        1 => ['null_rnaseq_columns'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'null_rnaseq_columns',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+        -parameters => {
+          db_conn => $self->o('rnaseq_db'),
+          sql => [
+            'UPDATE dna_align_feature SET external_db_id = NULL',
+          ],
+        },
+        -rc_name    => 'default',
+        -flow_into => {
+                        1 => ['rnaseq_set_align_type'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'rnaseq_set_align_type',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+        -parameters => {
+          db_conn => $self->o('rnaseq_db'),
+          sql => [
+            'UPDATE dna_align_feature SET align_type = "ensembl"',
+          ],
+        },
+        -rc_name    => 'default',
+        -flow_into => {
+                        1 => ['rnaseq_gene_sanity_checks'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'rnaseq_gene_sanity_checks',
+        -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+        -parameters => {
+                         target_db => $self->o('rnaseq_db'),
+                         sanity_check_type => 'gene_db_checks',
+                         min_allowed_feature_counts => get_analysis_settings('Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+                                                                             'gene_db_checks')->{'rnaseq_final'},
+                       },
+
+        -rc_name    => '4GB',
+        -flow_into => {
+                        1 => ['create_bam_file_job'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'create_bam_file_job',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+        -parameters => {
+          inputcmd => 'cd #working_dir#; ls *.bam',
+          column_names => ['bam_file'],
+          working_dir => $self->o('merge_dir'),
+        },
+        -rc_name => 'default',
+        -flow_into  => {
+          '2->A' => ['bam2bigwig'],
+          'A->1' => ['concat_md5_sum'],
+        },
+      },
+
+
+      {
+        -logic_name => 'bam2bigwig',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -parameters => {
+	  deeptools_bamcoverage => $self->o('deeptools_bamcoverage_path'),
+          cmd => 'TMPDIR=#working_dir# ; #deeptools_bamcoverage# --numberOfProcessors 4 --binSize 1 -b '.catfile('#working_dir#','#bam_file#').' -o '.catfile('#working_dir#','#bam_file#').'.bw',
+          working_dir => $self->o('merge_dir'),
+        },
+        -rc_name => '10GB_multithread',
+	-hive_capacity => $self->hive_capacity_classes->{'hc_very_low'},
```

Comment:
I know I said no "optimisation". I don't see the reason for this -hive_capacity setting, the resource requirement will probably already limit the number of jobs people will have.
We can keep it but that mean you have to bring back the hive_capacity_classes method in your config file or add the method to the HiveBaseConfig.
perl -c is not clever enough for method on an object. If I had given you the init command previously, it should have compained while initialising.
If you keep it it would be great if you can remove the tab and use space instead to keep indentation

Reply:
I agree with removing, this is best handled externally.